playbook/app/pb_kits/playbook/pb_image/image.rb

```diff
@@ -16,7 +16,7 @@ class Image
       prop :url
 
       def classname
-        generate_classname("pb_image_kit lazyload blur_up", size_class) + rounded_class
+        generate_classname("pb_image_kit#{size_class} lazyload blur_up") + rounded_class
```

Comment:
Just a little re-ordering here solved the issue with classnames being out of order and generally wrong.